en/sql/schema/view_stmt.rst

```diff
@@ -294,15 +294,55 @@ You can drop a query defined in the query specification by using the **DROP QUER
 COMMENT Clause
 --------------
 
-You can change a view's comment with **COMMENT** clause of **ALTER VIEW** syntax.
+You can change a view's comment and column comments with **COMMENT** clause of **ALTER VIEW** syntax.
```

Comment:
You can change a view's comment and column comments with
-->
You can change a view's comment,  columns' comment, or attributes' comment with ...

Reply:
Thank you. Fixed in '5228902' commit.